code/hud/hudtarget.cpp

```diff
@@ -5533,6 +5533,16 @@ void HudGaugeWeaponEnergy::initBitmaps(char *fname)
 	Energy_bar.first_frame = bm_load_animation(fname, &Energy_bar.num_frames);
 	if ( Energy_bar.first_frame < 0 ) {
 		Warning(LOCATION,"Cannot load hud ani: %s\n", fname);
+	} else {
```

Comment:
Yeah, that is often the case, but my method will always work, for any PR, even with some pretty messy merges (as long as it doesn't think there are commits in the PR that aren't actually part of the PR).  I'm pretty sure I posted the command line for a GUI would generate a diff like you get on Github, BitBucket, etc.  Or it's the closest I've found to it so far.